batcher/aligned-batcher/src/main.rs

```diff
@@ -46,7 +46,7 @@ async fn main() -> Result<(), BatcherError> {
     // spawn task to listening for incoming blocks
     tokio::spawn({
         let app = batcher.clone();
-        async move { app.listen_new_blocks().await.unwrap() }
+        async move { app.listen_new_blocks().await }
```

Comment:
This seems to ignore the errors, is that correct?

Reply:
This change seems strange, the unwrap should stay, app can't recover from this

Reply:
We can change it to an expect to make it more clear that we cannot recover from this

Reply:
`expect()` added